src/Features/SupportEvents/UnitTest.php

```diff
@@ -13,7 +13,7 @@ public function receive_event_with_attribute()
         $component = Livewire::test(new class extends Component {
             public $foo = 'bar';
 
-            #[On('bar')]
+            #[BaseOn('bar')]
```

Comment:
Wouldn't it be better to use the `src/Attributes/On` for all those tests? Since users also will use those?